packages/ui-informed/tailwind.config.js

```diff
@@ -0,0 +1,8 @@
+/** @type {import('tailwindcss').Config} */
+export default {
+  content: ['./index.html', './src/**/*.{js,ts,jsx,tsx}'],
```

Comment:
we`re probably going to need to initialize the @utima/ui tailwind plugin, in order for it to load all the styles properly.